src/Schemes/Session.js

```diff
@@ -80,7 +80,10 @@ class SessionScheme extends BaseScheme {
      */
     if (rememberToken && duration) {
       this._ctx.response.cookie(this.rememberTokenKey, rememberToken, {
-        expires: new Date(Date.now() + duration)
+        expires: new Date(Date.now() + duration),
+        httpOnly: true,
+        sameSite: true,
+        path: '/'
```

Comment:
May I know, whey we are hardcoding `path` here and not reading it from the config file?

Reply:
this commit 39deaf0 did not pass the tests. So I hardcodded them.